doc/build/orm/tutorial.rst

```diff
@@ -783,6 +783,9 @@ Here's a rundown of some of the most common operators used in
         session.query(User.name).filter(User.name.like('%ed%'))
     ))
 
+ .. note:: for composite (multi-column) IN queries, group columns together
+    using :func:`.tuple_`
+
```

Comment:
this would need a short code example for sure, in the tutorial, people won't know how to figure this out